assets/js/components/ReportError.js

```diff
@@ -166,30 +182,35 @@ export default function ReportError( { moduleSlug, error } ) {
 
 	return (
 		<CTA title={ title } description={ description } error>
-			<div className="googlesitekit-error-cta-wrapper">
-				{ showRequestAccessURL && (
-					<Button href={ requestAccessURL } target="_blank">
-						{ __( 'Request access', 'google-site-kit' ) }
-					</Button>
-				) }
-				{ showRetry ? (
-					<Fragment>
-						<Button onClick={ handleRetry }>
-							{ __( 'Retry', 'google-site-kit' ) }
+			{ ! isViewOnly && (
```

Comment:
I think I had misinterpreted the ACs when writing the IB, but I believe we do want to show this section for all users, but just remove the `Request access` CTA and retry actions for view-only users.

Could you remove this condition here, and update the `showRequestAccessURL` and `showRetry` boolean variables to only be `true` if `isViewOnly` is falsey, besides the existing conditions?